.github/workflows/release.yml

```diff
@@ -109,7 +109,7 @@ jobs:
           path: artifacts
 
       - name: Create Release
-        if: job.status == 'success' && contains(github.ref, 'refs/tags/v')
+        if: job.status == 'success' && contains(github.ref, 'refs/tags/0.')
```

Comment:
What happens when we go to 1.0? :)

Reply:
Fixed, master push has different ref so there is no need at all to check last section of that.